src/public/scss/changelist.scss

```diff
@@ -11,6 +11,14 @@
     color: $gray-600;
   }
 
+  &__list-actions {
+    display: flex;
+
+    .btn {
+      margin-left: 0.5rem;
```

Comment:
According to what I read in the rest of the file the convention is bottom + right, can you change it to a right margin elsewhere ?

Reply:
Seen together, there is no standard in the code for now, we'll that later